tracer/src/Datadog.Trace/Tagging/TagsList.cs

```diff
@@ -127,13 +127,6 @@ private int WriteTags(ref byte[] bytes, int offset, Span span)
             // write "well-known" span-level tags (from properties)
             count += WriteAdditionalTags(ref bytes, ref offset);
 
-            if (span.IsRootSpan)
-            {
-                // write trace-level tags
-                var traceTags = span.Context.TraceContext?.Tags;
-                count += WriteTraceTags(ref bytes, ref offset, traceTags);
-            }
```

Comment:
you can keep this if you want

Reply:
Yeah, thanks. Restored in e86d56985de3b790ba678c8b05e7e21a6e287f1c.